src/curated-corpus/components/RemoveItemForm/RemoveItemForm.validation.tsx

```diff
@@ -41,7 +41,8 @@ export const validationSchema = yup
       obj[RemovalReason.PublishDate] ||
       obj[RemovalReason.SetDiversity] ||
       obj[RemovalReason.TimeSensitive] ||
-      obj[RemovalReason.TopicDiversity]
+      obj[RemovalReason.TopicDiversity] ||
+      obj['otherReason']
```

Comment:
just to check - this is the "Other" checkbox? or is this the free text field? if the former, all good. if the latter, i think we need at least one checkbox checked?

Reply:
@jpetto i believe the curators want to be able to enter a reason without specifying a check-mark. so either one of the checkboxes needs to be selected or at least the "other" reason needs to be provided in order to submit the form. both fields are optional in the mutation so shouldn't be an issue for the back-end?